### DISCUSSION TITLE
How can I run koreader in verbose mode in pocketbook?

### DISCUSSION THREAD

**Discussion by Thegoban** [koreader/koreader#8709] [Q&A] [answered]
It's all said, that is:

How can I run koreader in verbose mode in pocketbook?

Thank you.

**Comment by Frenzie** [comment, marked as answer]
The easiest way is from the file browser -> tools -> developer options.

**Reply by Thegoban** [reply]
Wow... it wasn't so hidden, how I missed that!

thank you ;)

**Reply by Thegoban** [reply]
Thank you a lot!, solved
